thunder/__init__.py

```diff
@@ -183,11 +183,11 @@ def __version__():
 if nvfuser_executor:
     add_default_executor(nvfuser_executor)
 
-if sdpa_executor:
-    add_default_executor(sdpa_executor)
+# if sdpa_executor:
+#     add_default_executor(sdpa_executor)
 
-if cudnn_executor:
-    add_default_executor(cudnn_executor)
+# if cudnn_executor:
+#     add_default_executor(cudnn_executor)
```

Comment:
Investigating why SDPA is passed to cudnn by default in nvfuser tests as well where the executor is specified.

Reply:
I think this should be done in a different PR.

Reply:
These lines will be restored, this was for verifying nvFuser API.
Since cudnn, sdpa executor are always used as executors even when the executor list is specified, SDPA is not given to nvFuser. I will fix that before putting up this PR for review and merge.

Reply:
Not trying to rush it, but is this PR ready for review yet?